pkg/horoscope/horoscope.go

```diff
@@ -126,17 +127,39 @@ func (h *Horoscope) Next(round uint, verify bool) (benches *Benches, err error)
 			"cost":     fmt.Sprintf("%vms", cost.Values),
 			"hints":    plan.Hints,
 		}).Infof("complete execution plan%d", plan.Plan)
+
+		if verify {
+			for _, set := range sets {
+				if !testOracle.Equal(set) {
+					benches.VerifiedFail = true
+					err = fmt.Errorf("results mismatch in plan(%d)", plan.Plan)
+					return
+				}
+			}
+		}
 	}
+
 	if verify {
-		err = verifyQueryResult(originList, rowsSet)
-		if err != nil {
-			panic(fmt.Sprintf("a critical error occurred for query %s: %v", qID, err))
+		for _, exec := range h.differentialExecs {
+			var results []executor.Comparable
+			_, results, err = RunSQLWithTime(exec, benches.Round, benches.DefaultPlan.SQL, benches.Type)
+			if err != nil {
+				return
```

Comment:
Why not try next differential DB here?

Reply:
It's difficult to deal with multiple mismatch errors... One mismatch is enough to report an error